requirements.txt

```diff
@@ -7,3 +7,5 @@ lxml==3.6.0
 PyYAML==3.11
 docutils==0.12
 python-magic==0.4.11
+PyJWT==1.7.1
```

Comment:
use `PyJWT[crypto] ~= 1.7.1` here and drop `cryptography`. The `[crypto]` includes all needed requirements.